Source/WebInspectorUI/UserInterface/Views/Variables.css

```diff
@@ -434,7 +434,7 @@ body {
         }
     }
 
-    &:not(.docked) {
+    &.mac-platform:not(.docked) {
```

Comment:
Drive-by fix follow-up from https://github.com/WebKit/WebKit/pull/35640#discussion_r1813994123 because I landed that patch before addressing this nit.